rbx_xml/src/conversion.rs

```diff
@@ -27,6 +27,10 @@ impl ConvertVariant for Variant {
         target_type: VariantType,
     ) -> Result<Cow<'_, Self>, String> {
         match (value.borrow(), target_type) {
+            (Variant::Int32(value), VariantType::Int64) => Ok(Cow::Owned((*value as i64).into())),
+            (Variant::Float32(value), VariantType::Float64) => {
+                Ok(Cow::Owned((*value as f64).into()))
+            }
```

Comment:
Can we use `i64::from` and `f64::from` so it's more clear no loss can occur in these conversions?